src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,60 @@
+package duke;
+
+import java.io.IOException;
+
+import duke.command.Command;
+
+/**
+ * Represents an instance of the Duke program.
+ */
+public class Duke {
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    /**
+     * Constructor for an instance of Duke.
+     * Contains a Ui for interacting with user,
+     * Storage for saving and retrieving tasks from a file,
+     * and Tasklist for storing the tasks.
+     * Catches any errors thrown when the program is running.
+     * @param filePath
```

Comment:
Should there be a new line between your description and @param line? 
Should there be no punctuation after each parameter description?